db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 Role.create(name: 'redactor', permission_attributes: { can_lock_user: false, can_edit_role: false, can_set_role: false, can_read: true, can_edit: true })
 Role.create(name: 'reviewer', permission_attributes: { can_lock_user: false, can_edit_role: false, can_set_role: false, can_read: true, can_edit: false })
 FactoryBot.create_list(:tag, 20)
+FactoryBot.create_list(:user, 5)
```

Comment:
maybe you can create users with different roles, known to us email and passwords?
Wouldn't it be nice and convenient for team?